tools/ManifestSignerClient/Readme.md

```diff
@@ -0,0 +1,44 @@
+# Manifest Signer Client
+
+Manifest Signer Client is a tool that will be used for the Manifest Trust feature. The deployment manifest JSON will be signed using the tool Manifest Signer client and the signatures will be verified by the IoT Edge Runtime once the signed deployment manifest is deployed.
+
+Manifest Signer client gives a signed deployment JSON file as an end output. 
+
+### Step 1: Certificate Generator
+Manifest signer client needs a root Certificate Authority and a signer cert. To achieve that Certificate Generator scripts will be used. There are two scripts in the CertificateGenerator folder
+1. Gen_RootCA_with_Signer_ECDsa.bat
+2. Gen_RootCA_with_Signer_RSA.bat
+
+##### Set of choices in DSA algorithms
+1. The first decision to make is which DSA algorithm schemes to choose. We highly recommend using ECDSA as it generates smaller signatures. Choose the scripts accordingly with the corresponding labelled script. 
+2. Next choice is the different algorithmic parameters. For ECDsa, there are three EC parameters to choose from secp521r1 , secp384r1, prime256v1. Edit the parameter `ROOT_KEY_ALGO` and `SIGNER_KEY_ALGO`. Default value for root CA is `scep521r1` and signer cert is `prime256v1`
+3. Next choice is the different SHA algorithm. For RSA and ECDsa, there are three choices of SHA algorithm to choose from  SHA256, SHA384, SHA512. Recommendation is to use larger keys for root CA and smaller keys for signer certs. Edit the parameter `ROOT_SHA_ALGORITHM` and `SIGNER_SHA_ALGORITHM`. Default value is `SHA256`
+3. The file names of the root CA, root private key, signer cert, signer private key are also editable in the script. 
+
+Once the variables are set in the script. Run the script in a `Powershell` and the following files will be generated as a result.
```

Comment:
Powershell is a windows only tool. Can one run through bash which is available on both linux and windows? I didn't see any specific dependency on powershell in these scripts.